core/src/main/scala/org/apache/spark/h2o/backends/SharedBackendUtils.scala

```diff
@@ -60,6 +63,14 @@ private[backends] trait SharedBackendUtils extends Logging with Serializable {
       AzureDatabricksUtils.setClientCheckRetryTimeout(conf)
     }
 
+    if (conf.isInternalSecureConnectionsEnabled) {
```

Comment:
Just a question: Is this executed only on a client? If so, can you point me to the place where it's set for workers?

Reply:
Yup, it is executed only on client -> we create the configuration object on the client and then distribute the files to spark